changelog.asc

```diff
@@ -2,6 +2,11 @@
 :sectanchors:
 = AnkiDroid Changelog
 
+== Version 2.8.5 (2018-10-05)
+* Fix widget crash on older devices (pre-Ice Cream Sandwich, Android 4.0.3 / API15)
+* Limit duplicate crash reports from same device
```

Comment:
Don't write changes that don't affect users

Reply:
Depends on the user - I think we were sending probably 100MB a month from some users, but they'd probably never know it and it was only the repetitive crashers like the widget users. Deleted it either way